Dockerfile

```diff
@@ -1,4 +1,8 @@
-FROM python:3.7-alpine3.8
+ARG BASE_IMAGE_VERSION
+
+FROM python:${BASE_IMAGE_VERSION}
+
+ARG POETRY_VERSION
```

Comment:
Das würde ich nach oben zum anderen ARG ziehen.

Reply:
Das habe ich versucht und musste leider feststellen, dass das nicht funktioniert...
Die ARGs über dem FROM sind nutzbar in dem FROM, wenn man innerhalb des Builds (alles unterhalb vom FROM) die Variable nutzen möchte, muss dies nach dem FROM definiert werden, da diese bei jedem FROM zurückgesetzt werden.